src/redux/actions/questionsActions/getQuestions.js

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+import { GET_QUESTIONS } from "../actionTypes.js";
+
+export function getProductQuestions(id) {
+  return function(dispatch) {
+    return axios
+      .get(`http://3.134.102.30/qa/${id}`)
+      .then(data => {
+        console.log("*** Sucessfully Get-ted Question Data From Server");
+        // console.log(data);
+        dispatch({
```

Comment:
Nice object literal use for inside dispatch. Are you dispatching this action numerous times? May be worth making a function that could be reused.

Reply:
I will look into that actually, i didn't consider that when constructing this request/dispatch